src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private final double katetA;
+    private final double katetB;
+    private String color;
```

Comment:
Since `color` is not changed after the object is constructed, it should also be marked as `final` to ensure the immutability of the class.